scripts/zones/Temple_of_Uggalepih/npcs/_mf6.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+-- Area: Temple of Uggalepih
+--  NPC: _mf8 (Granite Door)
+-- Note: Opens with Uggalepih Key
+-- !pos -208 -1.89 -20
+-----------------------------------
+local ID = require("scripts/zones/Temple_of_Uggalepih/IDs")
+require("scripts/globals/npc_util")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHasExactly(trade, xi.items.UGGALEPIH_KEY) then
+        player:confirmTrade()
+        player:messageSpecial(ID.text.YOUR_KEY_BREAKS, 0, xi.items.UGGALEPIH_KEY)
+        npc:openDoor(6.5)
```

Comment:
I am reasonably sure this does not actually give you the .5 and that either 6 or 7 is close enough.

(7 happens to be the default because a majority of retail doors do that length - at least as far as we can tell with japan latency)

Reply:
To echo that, 7 is also the default if no parameter is provided